cdap-validation/bin/input.py

```diff
@@ -0,0 +1,66 @@
+
+import getopt
+import sys
+import test_helpers as helpers
+
+##### INPUT FUNCTIONS #####
+
+def process_input(argv):
+## Calls the functions below
+# take input
+#    print blah
+    input_vars = process_params(argv)
+    return input_vars
+
+def process_params(params):
+    input_vars = {}
+    input_vars['verbose'] = 0
+    try:
+        opts, args = getopt.getopt(params, 'hc:m:u:U:vd', ['help', 'cluster=', 'module=', 'user=', 'uri=', 'verbose'])
```

Comment:
optparse: "Deprecated since version 2.7: The optparse module is deprecated and will not be developed further; development will continue with the argparse module."
argparse is mostly 2.7+. For 2.6, It requires installation via pip and we are doing our best to avoid having to install any additional packages.

About the option parsing, I originally had this in the main script. I moved it here to be consistent with the design. The thinking in the design was separate all input processing from the test framework, so the framework can just grab the JSON input object and whatever else it needs and work from there.

At least two more functions will be added here:
- validate input
- write input/parameter info to JSON object

Are you thinking that the input functions don't need to be separated out from the main script or testing.py?